.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -31,6 +31,12 @@ body:
       description: A clear description of what you expected to happen.
     validations:
       required: true
+  - type: textarea
+    attributes:
+      label: System Info
+      description: Output of `npx envinfo --system --npmPackages @stencil/core --binaries --browsers`
+      render: shell
```

Instead of using `envinfo`, can we ask folks to just provide what they think is the most important info?
```suggestion
      description: Operating system & version, npm, node, and browser(s) & version(s) (if applicable)
```

---

OK